pkg/framework/framework.go

```diff
@@ -256,6 +258,11 @@ func (f *Framework) BeforeEach(contexts ...string) {
 		if !f.SkipNamespaceCreation {
 			ctx.AddNamespacesToDelete(namespace)
 		}
+
+		options := kubeinformers.WithNamespace(namespace.GetName())
+		informerFactory := kubeinformers.NewSharedInformerFactoryWithOptions(kubeClient, time.Second*30, options)
+		EventHandler := events.EventHandler{}
```

Comment:
Isn't it supposed to be stored into the f.EventHandler instead?
One more related question. Maybe it would be better to set the event handler
into the ContextData type, like ctx.EventHandler, so we could have one instance
for each context (when using multiple contexts).

Reply:
That is true, I'll move EventHandler to ctx.